apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,275 @@
+				$('#confirmDeleteButton').on('click', function () {
+					// Set the __action value to "delete_override"
+					$('form[name="delete-override-form"] input[name="__action"]').val('delete_override');
+
+					// Submit the form
+					$('form[name="delete-override-form"]').submit();
+				});
+		});
+
+	</script>
+	<%- include('../partials/navbar'); %>
+	<main id="content" class="container-fluid">
+		<div class="card mb-4">
+			<div class="card-header bg-primary text-white d-flex align-items-center justify-content-between">
+				Access Overrides - <%= assessment.title %>
+				<button type="button" class="btn btn-light" data-toggle="modal" data-target="#addPolicyModal">
+					Add <i class="fas fa-plus"></i>
+				</button>
+			</div>
+			<div class="modal fade" id="addPolicyModal" tabindex="-1" role="dialog" aria-labelledby="addPolicyModalLabel" aria-hidden="true">
+				<div class="modal-dialog" role="document">
+					<div class="modal-content">
+						<div class="modal-header">
+							<h5 class="modal-title" id="addPolicyModalLabel">Add New Override</h5>
+							<button type="button" class="close" data-dismiss="modal" aria-label="Close">
+								<span aria-hidden="true">&times;</span>
+							</button>
+						</div>
+						<div class="modal-body">
+							<form id="add-new-override" method="POST">
+								<input type="hidden" name="__action" value="add_new_override">
+								<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+								<input type="hidden" class="form-control" id="current_student_uid" name="current_student_uid" value="<%= user.uid %>" required readonly>
+								<% if (!assessment.group_work) { %>
+									<div class="form-group">
+										<label for="student_uid">Student UID</label>
+										<input type="text" class="form-control" id="student_uid" name="student_uid" placeholder="student@example.com" required>
+									</div>
+								<% } else { %>
+									<div class="form-group">
+										<label for="group_id">Group Name</label>
+										<input type="text" class="form-control" id="group_name" name="group_name" required>
+									</div>
+								<% } %>
+								<div class="form-group">
+									<label for="start_date">Start Date</label>
+									<input type="datetime-local" class="form-control" id="start_date" name="start_date" required>
+								</div>
+								<div class="form-group">
+									<label for="end_date">End Date</label>
+									<input type="datetime-local" class="form-control" id="end_date" name="end_date" required>
+									<div id="end_date_error" class="invalid-feedback" style="display: none; color: red;"></div>
+								</div>
+								<div class="form-group">
+									<label for="credit">Credit</label>
+									<div class="input-group">
+									<input type="number" class="form-control" id="credit" name="credit" min="0" required>
+									<div class="input-group-append">
+										<span class="input-group-text">%</span>
+									</div>
```

Comment:
Please use proper indentation here. You can fix this when you convert tabs to spaces.

Reply:
Fixed in e412baec6